invokeai/frontend/web/src/app/types/invokeai.ts

```diff
@@ -49,6 +49,7 @@ export type AppConfig = {
   disabledFeatures: AppFeature[];
   disabledSDFeatures: SDFeature[];
   canRestoreDeletedImagesFromBin: boolean;
+  nodesDenylist: string[];
```

Comment:
What do you think about making this an `allowList` instead so that we have to explicitly allow nodes to be in there? Otherwise we will be on the defense if there are nodes we want to exclude

Reply:
I'll add an allowlist so the behavior mirrors that of the backend filtering.